cognite/seismic/protos/types.proto

```diff
@@ -213,6 +214,18 @@ message Geometry {
     }
 }
 
+// User-provided coverage override for surveys
+message SurveyCoverageOverride {
+    message NoOverride { };
+    oneof override {
+        // Overrides survey coverage with the provided geometry
+        Geometry coverage_override = 1;
+        // Specifies that coverage should not be overridden, but computed from
+        // the seismicstores in the survey
+        NoOverride no_override = 2;
```

Comment:
Will this be what it defaults to anyway if survey coverage override is none?

Reply:
Yes, but it is useful to have a separate variant for this, so we can edit it away.

An EditSurveyRequest with `coverage_override = None` is interpreted as "do nothing, leave it as it is". So in order to remove the override, we need to set it explicitly as `NoOverride`.

Cf. `SurveyGridTransformation`, where the `DeduceFromTraces` variant plays the same role.

Reply:
That makes sense, thanks